### DISCUSSION TITLE
What means product = [""] in `params.toml` section [seo] ?

### DISCUSSION THREAD

**Discussion by cnichte** [thuliteio/doks#1149] [Q&A] [answered]
I have entered my section `publications` as a test, because these are things I normally sell. 

Now I get an error during the build:

```
ERROR failed to process "/publications/applications/lifestories/index.html": "/var/folders/_t/dcbrkh657xqbpy8yy0gbmbn00000gn/T/hugo-transform-error3865702583:207:44": JSON parse error: expected comma character or an array or object ending on line 207 and column 44
   43:         "price": "",
               ^
```

which is no wonder, because a comma is missing in the generated code, before "price or after `"priceCurrency": ""`

```
      "offers": {
        "@type": "Offer",
        "url": "/publications/applications/lefestories/",
        "priceCurrency": ""      <------------------------- 
        "price": "",
        "availability":  null 
      }
```

Have I stumbled across an error? What exactly do I have to do to get rid of the error? I think I'm missing some configuration?

Best regards - Carsten

**Comment by h-enk** [comment, marked as answer]
> Have I stumbled across an error?

Yes, you have. Will be fixed. For now, override `product.html` (copy `node_modules/@hyas/seo/layouts/partials/seo/schemas/product.html` and paste as `/layouts/partials/seo/schemas/product.html`) and add a trailing comma to line 45:

```html
        "priceCurrency": "{{ $c }}",
```
